thunder/src/util.cc

```diff
@@ -44,7 +44,12 @@ double get_hpwl(const ::vector<Net> &netlist,
             if (pos.y > ymax)
                 ymax = pos.y;
         }
-        hpwl += (xmax - xmin) + (ymax - ymin);
+        int exp = 1;
+        char* dist_exp = std::getenv("PNR_PLACER_EXP");
```

Comment:
Can you refactor this such that the option is set in the placer? This function gets called very frequently and getting env is not efficient.

Reply:
For instance, to make it easier to implement, you can have a static boolean variable that gets set in the placer, but read out here.

Reply:
Does this look better?